### DISCUSSION TITLE
What about inline editing list, select?

### DISCUSSION THREAD

**Discussion by mike-seger** [vitmalina/w2ui#2319] [Q&A] [answered]
Assuming that the ERROR in the current 2.x:
**ERROR: input types "list", "enum" and "file" are not supported in inline editing.**
https://w2ui.com/web/demos/#/grid/21
is just a matter of time until they will work.
It also seems that "select" does not work.
"combo" does work, but it has some usability issues. 

Would it make sense to add a tracked issue, to get this missing functionality onto the radar? Maybe someone from the community (including myself) will make suggestions or find time to fix this.

**Comment by mike-seger** [comment]
Answering myself:
According to https://github.com/vitmalina/w2ui/discussions/1955
overlays are a general headache and something like
[popper](https://popper.js.org/) might the basis of the overhaul for such components ...

**Comment by vitmalina** [comment, marked as answer]
Well, `list` is now working, see https://w2ui.com/web/demos/#/grid/21 and tooltips have been fully refactored and no longer a headache. See https://w2ui.com/web/demos/#/utils/15. I initially tried to integrate popper and I knew some w2ui users used it, but had really hard time to do it. Their API is not very clear. So, I decided it would be easier and cleaner to just refactor built-in tooltips and overlay and that work is now done
